include/RevThread.h

```diff
@@ -89,20 +85,19 @@ class RevThread {
   bool FindFD(int fd){ return fildes.count(fd); }
 
   // Get the threads
-  std::unordered_set<int> GetFildes(){ return fildes; }
+  const std::unordered_set<int>& GetFildes(){ return fildes; }
 
   // Register file operations
-  const RevRegFile* GetConstRegFile() const { return &RegFile; }
+  const RevRegFile* GetRegFile() const { return &RegFile; }
```

Comment:
I definitely like getting rid of the `GetConstRegFile` function... So overloading this won't be problematic? I didn't realize you could do this with `const` versions of functions but if so then this is great

Reply:
Yes, Just overload one function with `const` after the parameter list `(...)`, and it will apply to `const`-qualified objects (or non-`const` if a better overload isn't available). You don't have to name it differently. The `cv-qualifier` of `*this`, as well as its optional reference specifier( blank, `&` or `&&` in C++11 or later)  participate in overload resolution.